packages/subgraph/schema.graphql

```diff
@@ -246,3 +250,36 @@ type CollSurplusChange implements Change @entity {
   collSurplusChange: BigDecimal!
   collSurplusAfter: BigDecimal!
 }
+
+type LqtyStake @entity {
+  id: ID!
+  owner: User!
+  amount: BigDecimal!
+  changes: [LqtyStakeChange!]! @derivedFrom(field: "stake")
+}
+
+enum LQTYStakeOperation {
+  stakeCreated
+  stakeIncreased
+  stakeDecreased
+  stakeRemoved
+  gainsWithdrawn
+}
+
+type LqtyStakeChange implements Change @entity {
+  id: ID!
+  sequenceNumber: Int!
+  transaction: Transaction!
+  systemStateBefore: SystemState!
+  systemStateAfter: SystemState!
+
+  stake: LqtyStake!
+  operation: LQTYStakeOperation!
```

Comment:
This should be a stake-specific name, e.g. `stakeOperation`. The reason is that different types implementing the same interface can't have fields with the same name but different types. This works _for now_ because all the other Change implementations have type-specific names for "operation", but as soon as you try to add another subtype you run into trouble.